protos/class.proto

```diff
@@ -11,19 +11,21 @@ service PresenterClassService {
     rpc DeleteClass (ClassDeleteRequest) returns (Status);
 
     rpc StartClass (ClassStartRequest) returns (stream ChatMessage);
-    rpc DelegateQuizQuestion (ClassEvent) returns (Status);
+    rpc DelegateQuizQuestion (QuizQuestion) returns (Status);
+    rpc GetQuizQuestionStatistics (QuizQuestion) returns (stream QuizQuestionStatistics);
     rpc EndClass (ClassEndRequest) returns (Status);
 }
 
 service StudentClassService {
-    rpc JoinClass (JoinClassRequest) returns (stream ClassEvent);
+    rpc JoinClass (JoinClassRequest) returns (stream QuizQuestion);
     rpc SendMessageToPresenter (ChatMessage) returns (Status);
+    rpc AnswerQuizQuestion (QuizQuestionAnswer) returns (QuizQuestionResponse);
 }
 
 message Class {
```

Comment:
dorzucilbym tutaj moze pole `User teacher`, taka informacja moze byc przydatna - zeby polaczyc Zajecia z nauczycielem

Reply:
możemy od razu wyodrębnić typy user'a - jakiś enum student,teacher

Reply:
chociaż sam nie wiem czy to najlepszy pomysł przesyłanie za każdym razem tej informacji..